tests/admin/test-class-admin.php

```diff
@@ -41,5 +61,28 @@ public function test_get_admin_features_NOT_ON_post_edit() {
 		$this->assertEquals( $admin_features, $class_instance->get_admin_features() );
 	}
 
+	/**
+	 * Check if get_wpseo_pages returns all wpseo_pages
+	 *
+	 * @covers WPSEO_Admin::get_wpseo_pages
+	 */
+	public function test_get_wpseo_pages() {
```

Comment:
This test could be removed if the `$wpseo_pages` becomes a constant in `class-admin.php`